### PR TITLE
#5016 Add instructions on how to install development build to Chrome and Firefox

### LINKED ISSUE #5016: Add install instructions to README
Add instructions on how to install development build to Chrome and Firefox

chrome instructions from https://github.com/FlowCrypt/flowcrypt-browser/pull/5007#issuecomment-1469536299
```
Please follow those steps.

1. Open your Chrome browser and navigate to chrome://extensions/.
2. If not already enabled, toggle on the Developer mode switch located at the top-right corner.
3. Click on the Load Unpacked button.
4. Browse to and select the appropriate folder, either build/chrome-consumer or chrome-enterprise.
Once completed, the extension will be successfully installed and ready to use.
```

### PR DESCRIPTION
**Pull request by ioanmo226**
This PR added instructions on how to install development build to Chrome and Firefox

close #5016 // if this PR closes an issue

issue #0000 // if it doesn't close the issue yet

----------------------------------

**Tests** _(delete all except exactly one)_:
- Does not need tests (refactor only, docs or internal changes)

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [x] addresses the issue it closes (if any)
- [x] code is readable and understandable
- [x] is accompanied with tests, or tests are not needed
- [x] is free of vulnerabilities
- [x] is documented clearly and usefully, or doesn't need documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,41 +10,33 @@ Mock tests: ![Mock tests](https://flowcrypt.semaphoreci.com/badges/flowcrypt-bro
...
-[02:42:12] Finished 'chromeBuildSpacesToTabs' after 4.55 ms
-[02:42:12] Starting 'copyChromeToFirefox'...
-[02:42:12] Finished 'copyChromeToFirefox' after 323 ms
-[02:42:12] Starting 'copyChromeToFirefoxEditedManifest'...
-[02:42:12] Finished 'copyChromeToFirefoxEditedManifest' after 10 ms
-[02:42:12] Finished 'default' after 7.97 s
+```bash
+$ npm run build
 ```
+After running this command, you can find your built project in the build/chrome-consumer and build/firefox-consumer folders.
```
Comment (sosnovsky) [inline README.md:25]: Let's highlight folder names with `` for better readability, so it'll look like:

find your built project in the `build/chrome-consumer` and `build/firefox-consumer` folders.

File: README.md [hunk 2]
```diff
@@ -10,41 +10,33 @@ Mock tests: ![Mock tests](https://flowcrypt.semaphoreci.com/badges/flowcrypt-bro
...
+
+To load the extension in Google Chrome, please follow these steps:
+
+1. Open your Chrome browser and navigate to chrome://extensions/.
+2. If not already enabled, toggle on the Developer mode switch located at the top-right corner.
```
Comment (sosnovsky) [inline README.md:30]: highlight `Developer mode`

File: README.md [hunk 3]
```diff
@@ -10,41 +10,33 @@ Mock tests: ![Mock tests](https://flowcrypt.semaphoreci.com/badges/flowcrypt-bro
...
+3. Click on the Load Unpacked button.
```
Comment (sosnovsky) [inline README.md:31]: highlight `Load unpacked`

File: README.md [hunk 4]
```diff
@@ -10,41 +10,33 @@ Mock tests: ![Mock tests](https://flowcrypt.semaphoreci.com/badges/flowcrypt-bro
...
+4. Browse to and select the appropriate folder, either build/chrome-consumer or chrome-enterprise.
```
Comment (sosnovsky) [inline README.md:32]: highlight "`build/chrome-consumer` or `chrome-enterprise`"

File: README.md [hunk 5]
```diff
@@ -10,41 +10,33 @@ Mock tests: ![Mock tests](https://flowcrypt.semaphoreci.com/badges/flowcrypt-bro
...
-[02:42:12] Finished 'copyChromeToFirefoxEditedManifest' after 10 ms
-[02:42:12] Finished 'default' after 7.97 s
+```bash
+$ npm run build
 ```
+After running this command, you can find your built project in the build/chrome-consumer and build/firefox-consumer folders.
+
+To load the extension in Google Chrome, please follow these steps:
+
+1. Open your Chrome browser and navigate to chrome://extensions/.
```
Comment (sosnovsky) [inline README.md:29]: highlight `chrome://extensions/`

File: README.md [hunk 6]
```diff
@@ -10,41 +10,33 @@ Mock tests: ![Mock tests](https://flowcrypt.semaphoreci.com/badges/flowcrypt-bro
...
+2. If not already enabled, toggle on the Developer mode switch located at the top-right corner.
+3. Click on the Load Unpacked button.
+4. Browse to and select the appropriate folder, either build/chrome-consumer or chrome-enterprise.
+
+Similarly, to load the extension in Firefox, follow these steps:
 
-Now you can find your built project in `build/chrome-consumer` and `build/firefox-consumer`
+1. Open your Firefox browser and navigate to about:debugging.
```
Comment (sosnovsky) [inline README.md:36]: highlight `about:debugging`

File: README.md [hunk 7]
```diff
@@ -10,41 +10,33 @@ Mock tests: ![Mock tests](https://flowcrypt.semaphoreci.com/badges/flowcrypt-bro
...
+2. Click on the "This Firefox" tab.
```
Comment (sosnovsky) [inline README.md:37]: highlight `This Firefox`

File: README.md [hunk 8]
```diff
@@ -10,41 +10,33 @@ Mock tests: ![Mock tests](https://flowcrypt.semaphoreci.com/badges/flowcrypt-bro
...
+3. Click on the "Load Temporary Add-on" button located at the top-right corner.
```
Comment (sosnovsky) [inline README.md:38]: highlight `Load Temporary Add-on`

File: README.md [hunk 9]
```diff
@@ -10,41 +10,33 @@ Mock tests: ![Mock tests](https://flowcrypt.semaphoreci.com/badges/flowcrypt-bro
...
+4. Browse to and select the appropriate folder, either build/firefox-consumer or firefox-enterprise, and select the manifest.json file.
```
Comment (sosnovsky) [inline README.md:39]: highlight:
`build/firefox-consumer`
`build/firefox-enterprise`
`manifest.json`

File: README.md [hunk 10]
```diff
@@ -10,41 +10,33 @@ Mock tests: ![Mock tests](https://flowcrypt.semaphoreci.com/badges/flowcrypt-bro
...
 
 Printing debug data to test logs can be done using special `Debug` class:
```
Comment (sosnovsky) [inline README.md:41]: In Firefox we can also run extension with `npm run run_firefox` command, should be added to `README` too
I usually use it, as in this case it runs in separate Firefox and doesn't mix with production extension installed.

File: README.md (hunk 1 above)
Comment (ioanmo226) [reply README.md:25]: Yep, I missed highlighting. Thanks

**Review by sosnovsky — APPROVED** [review]

**Pull request merged**